### PR TITLE
Remove platform window handle at flutter_platform_view.h

### PR DESCRIPTION
**Pull request by bwikbs**
It will be replaced by ``FlutterDesktopGetWindow``

Related #209

### CODE REVIEW

**Comment by swift-kim** [comment]
Please also consider deprecating `FlutterRegisterViewFactory` and adding `FlutterDesktopRegisterViewFactory` (or just renaming it) if you're going to make a breaking change in the pltaform view API.

**Comment by bwikbs** [comment]
> Please also consider deprecating FlutterRegisterViewFactory and adding FlutterDesktopRegisterViewFactory (or just renaming it) if you're going to make a breaking change in the pltaform view API.

Of course. In this case, I guess it seems enough to change its name.

**Review by swift-kim — APPROVED** [review]
Thank you!

> BTW, This must have been pretty annoying you until now. Am I right? :smile:

Yeah, I was looking forward to a breaking API change in the platform view API ;)

File: shell/platform/tizen/public/flutter_platform_view.h [hunk 1]
```diff
@@ -19,8 +19,7 @@ class PlatformView {
   PlatformView(flutter::PluginRegistrar* registrar,
                int view_id,
                void* platform_window)
```
Comment (HakkyuKim) [inline shell/platform/tizen/public/flutter_platform_view.h:21]: `platform_window` can be removed from constructor.

Comment (bwikbs) [reply shell/platform/tizen/public/flutter_platform_view.h:21]: You're right!! Thanks! 👍

**Pull request merged**